server/server.go

```diff
@@ -782,7 +782,7 @@ func (s *Server) startSeccompNotifierWatcher(ctx context.Context) error {
 				})
 			}
 
-			metrics.Instance().MetricContainersSeccompNotifierCountTotalInc(ctr.Name(), usedSyscalls)
+			metrics.Instance().MetricContainersSeccompNotifierCountTotalInc(ctr.Name(), syscall)
```

Comment:
above we pull `notifier.UsedSyscalls`which got use usedSyscalls. I don't see us finding all of the syscalls that were reported. shouldn't we call this function for each syscall?

Reply:
This is called in a loop, every time a seccomp notification arrives on the channel:

https://github.com/cri-o/cri-o/blob/093d680dd32276457a62cb3615b45b01610e5030/server/server.go#L746-L750

`usedSyscalls` accumulates syscalls from each iteration in the same loop:

https://github.com/cri-o/cri-o/blob/093d680dd32276457a62cb3615b45b01610e5030/server/server.go#L764

So we do call the function for each syscall - we just incr the metric count for the specific syscall name received each iteration, ie:

| iteration | msg.Syscall() | metric values |
|----------|----------------|----------------|
| 0 | swapoff | `{name="...", syscall="swapoff"} = 1` |
| 1 | swapoff | `{name="...", syscall="swapoff"} = 2` |
| 2 | chroot | `{name="...", syscall="swapoff"} = 2`, `{name="...", syscall="chroot"} = 1` |